### ISSUE TITLE
Nice plugin, support vim please.

### ISSUE THREAD

**Issue by skywind3000** [APZelos/blamer.nvim#8]
Love it, could you please port it to vim ? This plugin will be more popular if it supports vim.

**Comment by Gee19** [comment]
I don't think Vim supports virtual text

**Comment by skywind3000** [comment]
@Gee19 , it's very simple.

Display a small popup window right after the last character of the current line.

**Comment by Gee19** [comment]
I'm sure @APZelos would accept a PR if it is simple ;)

**Comment by J0Miles** [comment]
Is there documentation anywhere on how to enable the popup feature for Vim? I get the error message
`[blamer.nvim] Needs popup feature.`

**Comment by skywind3000** [comment]
https://github.com/vim/vim/blob/master/runtime/doc/popup.txt

here is the documentation, it comes with vim 8.2, maybe you can get a latest vim by 

```
$ brew install vim
```

**Comment by J0Miles** [comment]
> https://github.com/vim/vim/blob/master/runtime/doc/popup.txt
> 
> here is the documentation, it comes with vim 8.2, maybe you can get a latest vim by
> 
> ```
> $ brew install vim
> ```

Worked like a charm after terminal restart. Thank you!

**Comment by georgesofianosgr** [comment]
Basic support for vim has been implemented #13 .

**Comment by Lakshamana** [comment]
Not working on my vim.
vim version:
```
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Apr 23 2021 22:01:35)
Correções incluídas: 1-2803
Compilado por Arch Linux
Versão enorme sem interface gráfica.  Recursos incluídos (+) ou não (-):
+acl               -farsi             +mouse_sgr         +tag_binary
+arabic            +file_in_path      -mouse_sysmouse    -tag_old_static
+autocmd           +find_in_path      +mouse_urxvt       -tag_any_white
+autochdir         +float             +mouse_xterm       -tcl
-autoservername    +folding           +multi_byte        +termguicolors
-balloon_eval      -footer            +multi_lang        +terminal
+balloon_eval_term +fork()            -mzscheme          +terminfo
-browse            +gettext           +netbeans_intg     +termresponse
++builtin_terms    -hangul_input      +num64             +textobjects
+byte_offset       +iconv             +packages          +textprop
+channel           +insert_expand     +path_extra        +timers
+cindent           +ipv6              +perl              +title
+clientserver      +job               +persistent_undo   -toolbar
+clipboard         +jumplist          +popupwin          +user_commands
+cmdline_compl     +keymap            +postscript        +vartabs
+cmdline_hist      +lambda            +printer           +vertsplit
+cmdline_info      +langmap           +profile           +virtualedit
+comments          +libcall           +python            +visual
+conceal           +linebreak         -python3           +visualextra
+cryptv            +lispindent        +quickfix          +viminfo
+cscope            +listcmds          +reltime           +vreplace
+cursorbind        +localmap          +rightleft         +wildignore
+cursorshape       +lua               +ruby              +wildmenu
+dialog_con        +menu              +scrollbind        +windows
+diff              +mksession         +signs             +writebackup
+digraphs          +modify_fname      +smartindent       +X11
-dnd               +mouse             +sound             +xfontset
-ebcdic            -mouseshape        +spell             -xim
+emacs_tags        +mouse_dec         +startuptime       -xpm
+eval              +mouse_gpm         +statusline        +xsmp_interact
+ex_extra          -mouse_jsbterm     -sun_workshop      +xterm_clipboard
+extra_search      +mouse_netterm     +syntax            -xterm_save
```
Revision detail
```
$ git rev-parse HEAD
> 1d5c4b5bcb0e25265ec6e112dcbd1226c4875648
```
